src/Razor/src/rzls/rzls.csproj

```diff
@@ -23,42 +21,8 @@
     </Content>
   </ItemGroup>
 
-  <!--
-    Technique for publishing multiple RIDs from
```

Comment:
Why are we also stopping publishing multiple RIDs? Is this PR solving two issues at the same time?

As far as I know, the reason we publish multiple RIDs is not _because_ we're self contained per se. For example, Roslyn is not self contained, but still publishes multiple RIDs.

Reply:
> As far as I know, the reason we publish multiple RIDs is not because we're self contained per se. For example, Roslyn is not self contained, but still publishes multiple RIDs.

I'm also not sure. It seems to be related to self contained but I'll try to figure out a bit more.

Reply:
> It seems to be related to self contained

If you're making that statement just due to the presence of the code here, I would definitely suggest we leave all of this alone without further data. I would expect this PR could just be the removal of the `SelfContained=true` that was on line 53 of the file